eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Utxo.hs

```diff
@@ -124,8 +129,9 @@ data UtxoEnv era
   = UtxoEnv
       SlotNo
       (Core.PParams era)
-      (Map (KeyHash 'StakePool (Crypto era)) (PoolParams (Crypto era)))
+      (Map (KeyHash 'StakePool (Crypto era)) (PoolParams (Crypto era))) -- DO WE NEED THIS, IN DPSTATE
       (GenDelegs (Crypto era))
+      (DPState (Crypto era)) -- (Map Ptr (Credential 'Staking (Crypto era)))
```

Comment:
I don't think we need this in the UTXO environment anymore.

Reply:
Good observation. It is now gone!